packages/libs/client-utils/vitest.integration.config.ts

```diff
@@ -3,6 +3,6 @@ import { defineConfig } from 'vitest/config';
 export default defineConfig({
   test: {
     include: ['src/**/*.int.test.ts'],
-    testTimeout: 60000,
+    testTimeout: 150000,
```

Comment:
Should be able to reduce this, due to instant mining sandbox

Reply:
can you please split the config for the functions need more times